src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static final int COMMAND_INDEX = 0;
```

Comment:
Good job using constants for your numbers! You could extend that to magic strings as well!